content/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,837 @@
+---
+linkTitle: Vertex Deformation for Vegetation Bending Tutorial
+title: Vertex Deformation for Vegetation Bending Tutorial
+description: A tutorial for add vegetation bending with the Atom rendered in the Open 3D Engine (O3DE).
```

Comment:
Put in "material type" and "shaders" since they're likely key search terms. Helps with SEO so people can find this page better. 
```suggestion
description: A tutorial for adding vegetation bending by creating custom a material type and authoring shaders with the Atom Renderer in the Open 3D Engine (O3DE).
```